config/locales/administrate.es.yml

```diff
@@ -6,6 +6,8 @@ es:
       destroy: Destruir
       edit: Editar
       show: Mostrar
+      new: Novaj
+      back: Reen
```

Comment:
I see 2 random non-spanish words in this excerpt of the file alone :)

I can commit a fix on Monday.